service/src/main/java/com/ge/predix/acs/attribute/connectors/ZoneConfiguration.java

```diff
@@ -0,0 +1,29 @@
+    package com.ge.predix.acs.attribute.connectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ZoneConfiguration {
+    @Autowired
+    private DefaultResourceAttributeReader defaultResourceAttributeReader;
+    @Autowired
+    private DefaultSubjectAttributeReader defaultSubjectAttributeReader;
+
+    private AttributeReader resourceAttributeReader;
+    private SubjectAttributeReader subjectAttributeReader;
+
+    public AttributeReader getResourceAttributeReader(final String zoneId) {
```

Comment:
When we are ready to persist zone configuration, zoneId will be used to read zone configuration from the database